src/main/kotlin/agallipo/main.kt

```diff
@@ -1,6 +1,6 @@
 package agallipo
 
-
+//created by alex :)
 fun main(args: Array<String>) {
     val pathname = "/Users/cxb0507/Desktop/example.json"
```

Comment:
If you add an example of the json file you can use it here, better than the route to you machine.